doc/en/components/pie-chart.md

```diff
@@ -164,6 +211,18 @@ Additionally you can use the `LegendItemTemplate` and `LegendItemBadgeTemplate`
 
 <div class="divider--half"></div>
 
+### How to Show Percentage in Pie Chart Legend
+
+<!-- TODO -->
+
+### $PlatformShort$ Advanced Pie Chart Show Percentage in Legend Example
+
+<!-- TODO -->
+<!-- <div class="sample-container loading" style="height: 500px">
+    <iframe></iframe>
+</div>
+<div class="divider--half"></div> -->
+
 ## Others Category
 Sometimes, the underlying data for the pie chart component will contain many items with small values. In this case, the Others category will permit automatic aggregation of several data values into a single slice
 
```

Comment:
Others Category

https://share.goabstract.com/9e5d9807-a47c-47c5-ae31-563f4f346f3d?collectionLayerId=1b13d46c-d5ce-4af4-b8fa-2cc82442d18b&mode=build

Since I can't comment form this point on I am giving here the link to the Explosion:

https://share.goabstract.com/9e5d9807-a47c-47c5-ae31-563f4f346f3d?collectionLayerId=84a8c5d3-c5a2-42a8-a1c8-e1ad1fcb9a77&mode=build

Reply:
And here's the Selection:

https://share.goabstract.com/9e5d9807-a47c-47c5-ae31-563f4f346f3d?collectionLayerId=cf040cfe-ada7-4bed-9277-736c74b1347d&mode=design

Reply:
I don't get the selection sample is the link correct because the design looks broken to me? @tosharky

Reply:
Yes it's correct. I've made it work the same as it is now. You can see it in the current topic that's online. It grays out selected areas of the chart. I don't really know how and when this is used in this way, but since it's different from the explosion I guess they are different use-cases.

Reply:
I think having two modes of selection, one that explodes and one that grays slices out for a pie chart hurts usability. I think you can think of a best of breed approach here and add it as a default. In the topic we show only what we believe is the best practice and if selection has some cool API or other stuff we show it under this best sample whatever it is.

Reply:
Well since I haven't seen this gray-ing selection anywhere else I think we can discard it then.